test/test_operators.jl

```diff
@@ -241,13 +241,16 @@ end
 @testitem "Symbolics" begin
     import IOCapture, DynamicPolynomials
 
+    𝒪 = Sunny.𝒪
+    𝒮 = Sunny.𝒮
```

Comment:
@kbarros is the idea that these are not public so it's ok? Confused why these lines are added here

Reply:
Yeah these were removed from the public API and will not be exposed to the users. More cleanups to follow. This PR is focused on switching to package extensions.